### PR TITLE
[#15] Updates the header and footer to match Tugboat.qa

### PR DESCRIPTION
**Pull request by helenasue**

### CODE REVIEW

**Comment by helenasue** [comment]
Tagging @bchavet for review

File: layouts/header.html [hunk 1]
```diff
@@ -2,12 +2,14 @@
     <div class="container">
         <div class="row navbar">
             <div class="col-xs-3">
-                <a href="https://tugboat.qa"><img class="logo" src="https://tugboat.qa/images/logo.svg"></a>
+                <a href="index.html"><img class="logo" src="https://tugboat.qa/images/logo.svg"><span class="accessible-hidden">Tugboat</span></a>
```
Comment (bchavet) [inline layouts/header.html]: This was linking back to tugboat.qa on purpose, so the header behavior is consistent.

**Comment by bchavet** [comment]
Looks great, except for the link back to tugboat.qa

File: layouts/header.html (hunk 1 above)
Comment (helenasue) [inline layouts/header.html]: Oops! Nice catch, Ben. I transferred all of the code over to make it the same and missed these. Thank you! :)

**Pull request merged**